content/courses/advanced-ada/parts/resource_management/access_types.rst

```diff
@@ -711,6 +711,98 @@ unconstrained designated type. In the :ada:`Show_Person` procedure, we declare
 the :ada:`P` object and specify the constraints of the allocated string object
 |mdash| in this case, a four-character string initialized with the name "John".
 
+.. admonition:: For further reading...
+
+    In the previous code example, we used an array |mdash| actually, a string
+    |mdash| to demonstrate the advantage of using discriminants as access
+    values, for we can use an unconstrained type as the designated subtype. In
+    fact, as we discussed
+    :ref:`earlier in another chapter <Adv_Ada_Indefinite_Subtype_Discriminant>`,
+    we cannot use indefinite subtypes as discriminants. Therefore, you wouldn't
```

Comment:
It's more restricted than not allowing indefinite types, we can only use discrete types (or access types, of course). You may have said that already and I missed it.

Reply:
Yes, I've just fixed it.